packages/sdk/src/protocols/light_push/light_push.ts

```diff
@@ -6,53 +6,51 @@ import {
   LightPushCore
 } from "@waku/core";
 import {
+  type CoreProtocolResult,
   Failure,
   type IEncoder,
   ILightPush,
   type IMessage,
+  type ISenderOptions,
   type Libp2p,
   type ProtocolCreateOptions,
   ProtocolError,
-  ProtocolUseOptions,
   SDKProtocolResult
 } from "@waku/interfaces";
 import { ensurePubsubTopicIsConfigured, Logger } from "@waku/utils";
 
-import { ReliabilityMonitorManager } from "../../reliability_monitor/index.js";
-import { SenderReliabilityMonitor } from "../../reliability_monitor/sender.js";
-import { BaseProtocolSDK } from "../base_protocol.js";
+import { DEFAULT_NUM_PEERS_TO_USE } from "../base_protocol.js";
 
 const log = new Logger("sdk:light-push");
 
-class LightPush extends BaseProtocolSDK implements ILightPush {
-  public readonly protocol: LightPushCore;
+const DEFAULT_MAX_ATTEMPTS = 3;
+const DEFAULT_SEND_OPTIONS: ISenderOptions = {
+  autoRetry: false,
+  maxAttempts: DEFAULT_MAX_ATTEMPTS
+};
+
+type RetryCallback = (peer: Peer) => Promise<CoreProtocolResult>;
 
-  private readonly reliabilityMonitor: SenderReliabilityMonitor;
+export class LightPush implements ILightPush {
```

Comment:
removed `BaseProtocolSDK` for now as it is not used
let's follow up in https://github.com/waku-org/js-waku/issues/2075